data/sitesEN.json

```diff
@@ -4965,15 +5156,16 @@
         "origin": "The Coffin of Andy and Leyley Fandom Wiki",
         "origin_base_url": "coffin-of-andy-and-leyley.fandom.com",
         "origin_content_path": "/wiki/",
-        "origin_main_page": "The_Coffin_of_Andy_%26_LeyLey_Wiki"
+        "origin_main_page": "The_Coffin_of_Andy_&_LeyLey_Wiki"
```

Comment:
Is it important that Main Page titles be URL-encoded? Not sure whether this should be handled on the data side or at runtime.

Reply:
In the current version of IWB, it seems that having it encoded in the JSON causes the redirection to fail.

This is not identified as the Fandom wiki's Main Page: 
https://coffin-of-andy-and-leyley.fandom.com/wiki/The_Coffin_of_Andy_%26_LeyLey_Wiki
While this invalid page title (corresponding to a URL-encoded version of %26) is considered the Fandom wiki's Main Page: https://coffin-of-andy-and-leyley.fandom.com/wiki/The_Coffin_of_Andy_%2526_LeyLey_Wiki

Reply:
I'll have to double-check, but I think they should be URL-encoded since that's how the links will appear in search engine link tags. Though ideally, IWB would normalize to a consistent format before comparing.

Reply:
Ah, I stand corrected!

Reply:
We do indeed decode at runtime in the `commonFunctionGetOriginArticle` function.

Reply:
Perhaps I should open an issue on this topic specifically, since it's technically a bug in the current version of IWB.

One option would be to store the Main Pages at their actual titles (i.e. `The Coffin of Andy & LeyLey Wiki` with spaces instead of underscores) and encode that at runtime based on the wiki's platform. (It would need to be platform-dependent though, since Fextralife and DokuWiki encode their page titles in URLs slightly differently to MediaWiki).